boards/cc1312-launchpad/dist/openocd.cfg

```diff
@@ -0,0 +1,11 @@
+source [find interface/xds110.cfg]
+
+transport select jtag
+gdb_memory_map enable
+gdb_flash_program enable
+
+source [find target/ti_cc26x2.cfg]
```

Comment:
Looks like this is generic enough to move it to move it to `./common/cc26x2_cc13x2/dist/openocd.cfg`.

The you can probably add `OPENOCD_CONFIG ?= $(RIOTBOARD)/common/cc26x2_cc13x2/dist/openocd.cfg` to `boards/cc1352-launchpad/Makefile.include` too.

Reply:
Thanks 😉!